tests/conftest.py

```diff
@@ -190,3 +190,45 @@ def cvxCRV(safe):
 def rari(dev):
     dev.init_rari()
     return dev.rari
+
+
+# balancer fixtures
+@pytest.fixture
+def balancer(dev):
+    dev.init_balancer()
+    return dev.balancer
+
+
+@pytest.fixture
+def bpt(dev):
+    return dev.contract(registry_addr.eth.balancer['50-BTC-50-WETH'])
+
+
+@pytest.fixture
+def staked_bpt(dev):
+    return dev.contract(registry_addr.eth.balancer['50-BTC-50-WETH-G'])
```

Comment:
i think it is good to start with moving protocol specific fixtures to `tests/<protocol>/conftest.py`.

i saw this being done here: https://github.com/Citadel-DAO/citadel-multisig/pull/6/files

Reply:
good idea